src/status_im/ios/core.cljs

```diff
@@ -31,7 +31,6 @@
          (.hide react/splash-screen))
        :component-did-mount
        (fn []
-         (notifications/request-permissions)
```

Comment:
why only this one, and not all?

Reply:
No reason, it's just setting up logic for when the user does say yes or no at the prompt - it's set up there in the android version so I left it in place and just moved the actual request to the home screen